### DISCUSSION TITLE
How to translude short up key?

### DISCUSSION THREAD

**Discussion by linonetwo** [TiddlyWiki/TiddlyWiki5#6166] [Q&A]
<img width="713" alt="截屏2021-11-03 下午11 21 29" src="https://user-images.githubusercontent.com/3746270/140089682-bb331004-dbdf-4e04-a8dc-be729aa85400.png" >

I hope I can add short cut info to the / menu, but seems it is not transcluable?

<img width="939" alt="截屏2021-11-03 下午11 21 45" src="https://user-images.githubusercontent.com/3746270/140089727-2d86d00f-cffb-48a2-8c65-c53241528520.png">

**Comment by saqimtiaz** [comment]
@linonetwo I am not sure that I understand what you are trying to do. Do you want to have an option in the menu shown in the screenshot, to trigger editor toolbar button functionality? Like Bold and Italic?

Is the menu shown in the screenshot based on the CommandPalette plugin?

**Reply by linonetwo** [reply]
That is https://github.com/Gk0Wk/TW5-CodeMirror-Enhanced , you can view the demo here https://gk0wk.github.io/TW5-CodeMirror-Enhanced/ .

That preview is editable in https://github.com/Gk0Wk/TW5-CodeMirror-Enhanced/blob/0a34fc8e3dcbcdbcdb7c26cc62e00087d1340acb/src/i18n/en-GB.json#L83 , can use wiki text. I hope I can add more info to the preview.
